daskernetes/tests/test_core.py

```diff
@@ -283,3 +282,36 @@ def test_scale_up_down(cluster, client):
         assert time() < start + 10
 
     assert set(cluster.scheduler.workers) == {b}
+
+
+def test_automatic_startup(image_name, loop, ns):
+    test_yaml = {
+        "kind": "Pod",
+        "metadata": {
+            "labels": {
+                "foo": "bar",
+            }
+        },
+        "spec": {
+            "containers": [{
+                "args": [
+                    "dask-worker",
+                    "$(DASK_SCHEDULER_ADDRESS)",
+                    "--nthreads",
+                    "1"
+                ],
+                "image": image_name,
+                "name": "dask-worker"
+            }]
+        }
+    }
+
+    with tmpfile(extension='yaml') as fn:
+        with open(fn, mode='w') as f:
+            yaml.dump(test_yaml, f)
+        config['worker-spec-path'] = fn
```

Comment:
Can we think of a more concise-and-descriptive name here?

Reply:
I like worker-template-path better, but is not more concise. Am ok with whatever you pick